### ISSUE TITLE
RSTDISBL with ArduinoIDE

### ISSUE THREAD

**Issue by burjab** [wholder/ATTiny10IDE#13]
Greetings,

I've been endeavoring to utilize the RSTDISBL fuse for the Attiny10 via the Arduino IDE. Regrettably, despite employing the `#pragma fuse RSTDISBL` directive in my code, Pin 6 continues to function as a reset pin even after compiling through the Arduino IDE. I'm seeking guidance on the correct procedure for achieving the desired outcome. Your assistance in this matter would be greatly appreciated.

**Comment by wholder** [comment]
I'm now a bit rusty on my attiny10 knowledge as I've also moved on to other projects, such as my new [MegaTinyIDE](https://github.com/wholder/MegaTinyIDE).  I personally did not make much use of the RSTDISBL fuse, but I do recall that I tested that function in ATTiny10IDE.  Howecer, you should be able to directly set this fuse using the "Read/Modify Fuse(s): feature in the "Actions" menu.  Have you tried this?  

BTW, what are you using to program the chip?

**Comment by burjab** [comment]
Thanks for getting back to me. Just wanted to let you know, I figured out the problem. When I used "#pragma fuses rstdisbl" in lowercase letters, it worked like a charm.

Also, I ran into a bit of a hiccup when I tried to build the code directly from AttinyIDE using Arduino. The program crashed and closed the IDE every time. So, I ended up building the code for Arduino and using ArduinoIDE for uploading. 

Hope this little tidbit helps out others too!
